pkg/kyverno/validate/command.go

```diff
@@ -33,6 +36,13 @@ func Command() *cobra.Command {
 				}
 			}()
 
+			if outputType != "" {
+				if outputType == "yaml" || outputType == "json" {
+				} else {
```

Comment:
What does this if clause do?

Reply:
we can print mutated policy in json and in yaml. The above if clause is to check the flag value.